packages/kv/README.md

```diff
@@ -1,4 +1,6 @@
-# @vercel/kv
+# @vercel/kv (deprecated)
+
+> [!CAUTION] > **`@vercel/kv` is deprecated**. Please use [the `@upstash/redis` package from Upstash instead](https://github.com/upstash/redis-js).
```

Comment:
<img width="1075" alt="Screenshot 2025-02-21 at 12 05 16" src="https://github.com/user-attachments/assets/1e2f39d0-3ff7-4cc2-b1e7-c298e87c573e" />

it seems the Note didn't render correctly


Should we add some info about the features in this package not avilable in `@upstash/redis`?

Reply:
Grrr. My editor formatter removed the new line, very strange. Updated! Thanks for checking.

> Should we add some info about the features in this package not avilable in @upstash/redis?

What had you in mind?

Reply:
- this package overrides `scan`, `hscanIterator`, `sscan`, `zscan`...not sure if upstash fixed that issue in the meantime, otherwise we could mention it in the deprecation note

Reply:
So you mean like an upgrade guide? I'll check in with the docs team to see where we are on that.